### PR TITLE
Suzuka Docker Compose Stack

### PR DESCRIPTION
**Pull request by l-monninger**
# Summary
- **RFCs**: $\emptyset$
- **Categories**: `cicd`

Sets up the Docker Compose service definition needed for the corresponding infrastructure stack in https://github.com/movementlabsxyz/infra/pull/33.

# Changelog

1. Adds [Suzuka service definition](https://github.com/movementlabsxyz/movement/tree/l-monninger/suzuka-docker-compose-stack/docker/compose/suzuka-full-node-monolith). 
2. Adds templates for [Monza](https://github.com/movementlabsxyz/movement/tree/l-monninger/suzuka-docker-compose-stack/docker/compose/monza-full-node-monolith) and [Suzuka](https://github.com/movementlabsxyz/movement/tree/l-monninger/suzuka-docker-compose-stack/docker/compose/suzuka-full-node-monolith) to be leveraged by the backend. 
3. Other changes are the results of syncing with miscellaneous CI branches which have made incremental improvements. 

# Testing
This has been tested e2e using Gali. See the links below and the associated PR in infra. 

# Outstanding issues
1. Currently the Gali and containers workflow is still such that you'd have to wait on an initial build because the deployment will try to fetch a container matching the commit hash at which the last Gali command was issued. 
2. We are still using the monolith of course.

### CODE REVIEW

**Comment by l-monninger** [comment]
gali

**Comment by l-monninger** [comment]
**Gali PR:** https://github.com/movementlabsxyz/infra/pull/33

**Comment by l-monninger** [comment]
gali

**Comment by l-monninger** [comment]
**Gali PR:** https://github.com/movementlabsxyz/infra/pull/33

**Comment by l-monninger** [comment]
gali

**Comment by l-monninger** [comment]
**Gali PR:** https://github.com/movementlabsxyz/infra/pull/33

**Comment by l-monninger** [comment]
gali

**Comment by l-monninger** [comment]
**Gali PR:** https://github.com/movementlabsxyz/infra/pull/33

**Comment by l-monninger** [comment]
gali

**Comment by l-monninger** [comment]
**Gali PR:** https://github.com/movementlabsxyz/infra/pull/33

File: docker/build/m1-da-light-node/Dockerfile
Comment (radupopa369) [inline docker/build/m1-da-light-node/Dockerfile:1]: By default all Docker fils that do not contain the command `USER nonroo` run as root. 
Do we need to run as root here?

File: docker/compose/suzuka-full-node-monolith/prometheus.yml
Comment (radupopa369) [inline docker/compose/suzuka-full-node-monolith/prometheus.yml:1]: The idea of docker-compose is to have all services in the same docker-compose file.
Any reason this prometheus file is separated?

File: docker/build/m1-da-light-node/Dockerfile
Comment (radupopa369) [reply docker/build/m1-da-light-node/Dockerfile:1]: With the current directory structure it is very difficult to follow how the docker images are being build and with with context

Comment (l-monninger) [reply docker/build/m1-da-light-node/Dockerfile:1]: Context is simply the top level of the repo.

**Review by radupopa369 — APPROVED** [review]

**Pull request merged**